src/com/googlecode/jmxtrans/model/output/OpenTSDBWriter.java

```diff
@@ -113,6 +113,11 @@ protected void startOutput() throws IOException {
 	@Override
 	protected void sendOutput(String metricLine) throws IOException {
 		try {
+			
+			if (isDebugEnabled() || log.isDebugEnabled()) {
+				log.debug("Sending to server {}:{} line={}", host, port,metricLine);
```

Comment:
Stop using Eclipse immediately and switch to Intellij. It is a far better IDE. =) I know that isn't what you want to hear, but trust me on this one... you won't look back.